bb/master/config.py

```diff
@@ -54,26 +50,29 @@ class Mode(Enum):
 BUILDBOT_TREE_STABLE_TIMER = None # Value "None" means that a separate build will be started immediately for each Change.
 BUILDBOT_TITLE = "Intel® Media SDK"
 
+POLL_INTERVAL = 10 # Poll Github for new changes (in seconds)
+
+WORKER_PASS = msdk_secrets.WORKER_PASS
+DATABASE_PASSWORD = msdk_secrets.DATABASE_PASSWORD
+DATABASE_URL = "postgresql://buildbot:%s@localhost/buildbot" % DATABASE_PASSWORD
```

Comment:
We can use something like f"postgresql://buildbot:{DATABASE_PASSWORD}@localhost/buildbot" in these cases. 
Just FYI.

Reply:
Hm, another words using of Python 3 style, ok. Will do.